packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -593,6 +593,7 @@ function EmptyTest() {
 
 export function DragExample(props?) {
```

Comment:
IMO it might make it easier to test if there was a story that allowed only copy or link operations. Either that or add a description to some of the stories that tell people to try and perform mouse drag and drop operations while holding the `option` or `control` key (aka the keyboard shortcuts for 'copy' and 'link')

Reply:
Just added a story that allows for `copy` and `link`, with a description.